packages/api/src/command/connected-user/get-connected-user.ts

```diff
@@ -81,10 +82,15 @@ export const getConnectedUserByTokenOrFail = async (
     },
   });
 
-  if (!connectedUser)
+  if (!connectedUser.length)
     throw new NotFoundError(`Could not find connected user with str matching token`);
 
-  return connectedUser;
+  if (connectedUser.length > 1)
+    capture.message(`Found multiple connected users with str matching token`, {
+      extra: { context: `getConnectedUserByTokenOrFail`, connectedUser },
```

Comment:
Thanks for doing that!

Can we also send the token (and rename from `str` to `token`) to Sentry, please? I think we should also set the level as `error`, what do you think?